src/lib/callApi.js

```diff
@@ -4,27 +4,38 @@ import { getLevelForContributionCount } from './Levels';
 const config = require('./config');
 
 let localData = [];
+/* eslint-disable global-require */
 try {
-  // eslint-disable-next-line global-require
   localData = require('./msf-mapswipe-users-export.json');
   // eslint-disable-next-line no-console
 } catch (e) { console.error(e); }
 
 // If eq 'dev' fetch data locally else (eq 'prod') fetch data from Firebase
-const SOURCE = 'dev';
+const SOURCE = 'prod';
```

Comment:
This seems like a really bad thing to be hard-coded to me. I'm not super familiar with the project, but can we source this from an env variable, or at least a config/constants file instead of hard-coded into a random `.js` file?

Reply:
Okay copy that